packages/application-shell/src/components/navbar/navbar.js

```diff
@@ -503,18 +515,20 @@ export class DataMenu extends React.PureComponent {
               ? menu.submenu.map(submenu => (
                   <RestrictedMenuItem
                     key={`${menu.key}-submenu-${submenu.key}`}
+                    keyOfMenuItem={`${menu.key}-submenu-${submenu.key}`}
```

Comment:
This should just be `submenu.key`. The string concatenation was just for ensuring that the react key is unique (probably not necessary).

```suggestion
                    keyOfMenuItem={submenu.key}
```